### DISCUSSION TITLE
Clarify what does "docker-compatible" mean

### DISCUSSION THREAD

**Discussion by spusuf** [Jip-Hop/jailmaker#23] [Q&A] [answered]
What does docker-compatible mean in the installation script as it is very vague. I saw mention to a network setting that is required for docker but is that all?

**Comment by Jip-Hop** [comment, marked as answer]
Docker compatibility is indeed vague but it's too much to explain in the installation script. You can read about it in the comments/code on these lines: https://github.com/Jip-Hop/jailmaker/blob/main/jlmkr.py#L185-L240.

**Reply by spusuf** [reply]
Is there anything preventing these variable definitions from being set by default and simplifying the jail creation process?

**Reply by Jip-Hop** [reply]
I want users who plan on using docker to enable this option explicitly, as it does some stuff you otherwise won't need.

I prefer your other suggestion, [to make the jailmaker create command configurable from the CLI](https://github.com/Jip-Hop/jailmaker/discussions/40).

As a quick hack you can modify your local copy of the python script to change the default behavior.

**Reply by spusuf** [reply]
Is there anything that would negatively affect performance or security?

**Reply by Jip-Hop** [reply]
Yes regarding security. You can find what it does in the source as mentioned before :)
